fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/SearchQueryRenderer.java

```diff
@@ -185,6 +187,10 @@ protected int getCodeSystemId(String codeSystemName) throws FHIRPersistenceExcep
         return this.identityCache.getCodeSystemId(codeSystemName);
     }
 
+    protected int getCanonicalId(String canonicalValue) throws FHIRPersistenceException {
```

Comment:
javadoc?
specifically I see the implementation returning -1 in some cases so it would be good to document what that means

Reply:
done